src/api/app/models/user_ldap_strategy.rb

```diff
@@ -220,7 +219,7 @@ def self.find_with_ldap(login, password)
       ldap_con.search(CONFIG['ldap_search_base'], LDAP::LDAP_SCOPE_SUBTREE, user_filter) do |entry|
         user = entry.to_hash
       end
-    rescue
+    rescue StandardError
```

Comment:
Should do. Just following general advice not to rescue everything:
http://blog.honeybadger.io/ruby-exception-vs-standarderror-whats-the-difference/

I believe that `rescue` with no arguments would rescue both Exception and StandardError which is not what we want.